.github/workflows/yocto.yml

```diff
@@ -76,13 +103,61 @@ jobs:
         cat tmp/work/core2-64-poky-linux/nnstreamer/*/temp/log.do_compile || true
         echo "::endgroup::"
 
+        echo "::group::Copy result"
+        mkdir -p ~/daily_build/logs
+        cp tmp/work/core2-64-poky-linux/nnstreamer/*/temp/log.do_configure ~/daily_build/logs/yocto_configure_log.txt
+        cp tmp/work/core2-64-poky-linux/nnstreamer/*/temp/log.do_compile ~/daily_build/logs/yocto_compile_log.txt
+        cp -r tmp/deploy ~/daily_build/deploy
+        echo "::endgroup::"
+
         sudo rm -rf tmp/
+
     - name: save yocto cache (main branch only)
       uses: actions/cache/save@v4
-      if: always() && github.ref == 'refs/heads/main'
+      if: steps.rebuild.outputs.rebuild == '1'
```

Comment:
It's desired to save caches only for main branch. How about keep the old line?

Reply:
Right. Let's save cache only for main branch. Thanks.